tests/testData.h

```diff
@@ -84,47 +84,83 @@ class DissolveSystemTest
     // Set up simulation ready for running, calling any additional setup function if already set
     template <int flags = 0> void setUp(std::string_view inputFile)
     {
-        dissolve_.clear();
-        if (!dissolve_.loadInput(inputFile))
-            throw(std::runtime_error(fmt::format("Input file '{}' failed to load correctly.\n", inputFile)));
-        if (rewriteCheck_)
+
+        if constexpr (Dissolve::toml_testing_flag || !(flags & TomlFailure))
         {
-            auto newInput = fmt::format("{}/TestOutput_{}.{}.rewrite", DissolveSys::beforeLastChar(inputFile, '/'),
-                                        DissolveSys::afterLastChar(inputFile, '/'),
-                                        ::testing::UnitTest::GetInstance()->current_test_info()->name());
-            if (!dissolve_.saveInput(newInput))
-                throw(std::runtime_error(fmt::format("Input file '{}' failed to rewrite correctly.\n", inputFile)));
-
-            dissolve_.clear();
-            if (!dissolve_.loadInput(newInput))
-                throw(std::runtime_error(fmt::format("Input file '{}' failed to reload correctly.\n", newInput)));
-        }
+            SerialisedValue toml;
+            {
+                CoreData other_;
+                Dissolve trial_{other_};
```

Comment:
These names are perhaps a bit too generic!  I might suggest:
```suggestion
                CoreData otherCoreData_;
                Dissolve otherDissolve_{otherCoreData_};
```

Reply:
I've gone ahead and renamed them `otherCoreData` and `otherDissolve`.  I dropped the trailing underscore because these aren't members of the class, but just variables in the function.